bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -6,21 +6,21 @@ num_users_updated = 0
 batch_size = 10000
 
 puts "Processing..."
-
-User.where.not(school_info: nil).where(user_type: "teacher").find_in_batches(batch_size: batch_size) do |batch|
+start = Time.now
+User.where.not(school_info_id: nil).where(user_type: "teacher").find_in_batches(batch_size: batch_size) do |batch|
   ActiveRecord::Base.transaction do
     batch.each do |user|
-      UserSchoolInfo.create!(
-        user_id: user.id,
-        school_info_id: user.school_info_id,
+      user.user_school_infos.where(school_info_id: user.school_info_id).first_or_create(
         start_date: user.created_at,
-        end_date: nil,
-        last_confirmation_date: user.created_at
+        last_confirmation_date: user.last_seen_school_info_interstitial.nil? ? user.created_at : user.last_seen_school_info_interstitial
       )
       num_users_updated += 1
+    rescue StandardError => e
+      p e
```

Comment:
do we want to inspect the entire error here, or are we okay with the contextual error you created on the next line (line 20)?